irods/exception.py

```diff
@@ -67,8 +69,60 @@ def __init__(self, name, bases, attrs):
             iRODSExceptionMeta.codes[attrs['code']] = self
 
 
+class Errno:
+    """Encapsulates an integer error code from the operating system
+       and provides a text representation.
+    """
+    def __init__(self,arg0,*args):
+        """Initializes an object with an integer error code arg0.
+           Further arguments are optional and ignored.
+        """
+        self.int_code = arg0
+    def __repr__(self):
+        e = self.int_code
+        try:
+            return self.__class__.__name__ + repr(tuple([e, errno.errorcode[e], os.strerror(e)]))
+        except Exception:
+            pass
+        return self.__class__.__name__ + repr(tuple([e,]))
+    def __int__(self):
```

Comment:
Let's add an empty line between these lines for readability.

Reply:
done!